pkg/v6/resource/chartconfig/create.go

```diff
@@ -26,9 +26,11 @@ func (r *Resource) ApplyCreateChange(ctx context.Context, obj, createChange inte
 		}
 
 		for _, chartConfigToCreate := range chartConfigsToCreate {
+			r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("creating chartconfig %s", chartConfigToCreate.Name))
```

Comment:
I was going mad with some debugging, this really helped. Happy to remove, but I found it useful.

Reply:
We should also have corresponding `created chartconfig` log. See https://github.com/giantswarm/fmt/blob/master/go/log_messages.md